concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -906,7 +906,7 @@ Hale-1973-1798-904	Food_and_Clothing-101	heat up roti	v	07b2.10	1341	HEAT UP
 Hale-1973-1798-905	Food_and_Clothing-102	heat	v	07b2.11	1341	HEAT UP
 Hale-1973-1798-906	Food_and_Clothing-103	cool	v	07b2.12	3068	COOL DOWN
 Hale-1973-1798-907	Food_and_Clothing-104	become soft (ripe)	v	07b2.13	178	RIPE
-Hale-1973-1798-908	Food_and_Clothing-105	season	v	07b2.14		
+Hale-1973-1798-908	Food_and_Clothing-105	season	v	07b2.14	691	SEASON
```

Comment:
`691 SEASON` is a noun in concepticon (https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L692). So I'd unmap.